tests/portals/test/test.portal

```diff
@@ -0,0 +1,3 @@
+[portal]
```

Comment:
We replaced `test.portal` with the `test.portal.in` template, which generates the portal file at build configuration time. You can drop this file.